src/app/clusters/refrigerator-alarm-server/refrigerator-alarm-server.cpp

```diff
@@ -136,6 +153,34 @@ EmberAfStatus RefrigeratorAlarmServer::SetStateValue(EndpointId endpoint, BitMas
     return status;
 }
 
+EmberAfStatus RefrigeratorAlarmServer::SetSupportedValue(EndpointId endpoint, const BitMask<AlarmMap> supported)
+{
+    EmberAfStatus status = EMBER_ZCL_STATUS_SUCCESS;
+    status               = Attributes::Supported::Set(endpoint, supported);
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    {
+        ChipLogProgress(Zcl, "Refrigerator Alarm: ERR: writing supported, err:0x%x", status);
+        return status;
+    }
+
+    ChipLogProgress(Zcl, "Refrigerator Alarm: Supported ep%d value: %" PRIx32 "", endpoint, supported.Raw());
```

Comment:
I see this in other places, however overall, why do we log status on every successful read? This seems quite spammy for logs.

Reply:
Should I make it `ChipLogDetails` because all other cluster implemetations have this logs or remove it?

Reply:
Detail seems more reasonable.

Reply:
Done